collectors/keywords.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+WHITELIST = ["Spring", "dotnet", "kernel"]
```

Comment:
If we want to have this discussion in our team, we should probably start getting the support from the top. Just recently I was specifically asked to implement a **blacklist** at https://issues.redhat.com/browse/OSIDB-1453 so what is different here? I personally do not understand what could be disrespectful about using established technical terms but I understand that I am probably not the most sensitive person in the world. Anyway we should either decide about not using it at all or we should be free to use it at will. Restricting it only somewhere makes no sense.

Reply:
Asked our manager and CCd you.

Reply:
There is actually a lot to do if we would follow these guidelines. 36 occurrences of the word blacklist in OSIDB codebase, usage of the `master` branch instead of `main` and of course there would be definitely more.

Reply:
The debate costs resources ATM.